design/cluster-api-provider-metal3/capm3-remediation-controller-improvement-proposal.md

```diff
@@ -0,0 +1,109 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# capm3-remediation-controller-improvement
+
+## Status
+
+implemented
+
+## Summary
+
+The original reboot remediation controller proposal [1] misses details on 
+on how remediation should actually be done. During implementation there was 
+some discussion if the Node should be deleted or not [2]. The decision was 
+to keep it simple and skip Node deletion.
+Skipping Node deletion has a big drawback though: workloads on unresponsive 
+Nodes won't be rescheduled quickly, because they are assumed to be still 
+running. Deleting the the Node signals that the workloads are not running 
+anymore, which results in quickly rescheduled workloads with less downtime.
```

Comment:
Would it make sense to move this part under `Motivation` section and leave line 25 only in the Summary (short and precise description of the change)?

Reply:
makes sense, done